src/Scripts/DockerLocalImage.ps1

```diff
@@ -72,9 +104,25 @@ if (-not (Test-Path "$dockerContext" -PathType Container)) {
     exit 1
 }
 
-$Expression = "Get-Content '$dockerDir\DockerFile' | docker build -t '$imageName' -f - '$dockerContext'"
-Write-Host $Expression
-Invoke-Expression $Expression | Write-Host
+if ($Wsl)
+{
+    $pathInfo = resolve-path $dockerDir
+    $drive = $pathInfo.Drive.ToString().ToLowerInvariant()
+    $dockerDir = $pathInfo.Path.Substring(3).Replace('\','/') 
+    $dockerDir = "/mnt/$drive/$dockerDir"
+    $pathInfo = resolve-path $dockerContext
+    $drive = $pathInfo.Drive.ToString().ToLowerInvariant()
+    $dockerContext = $pathInfo.Path.Substring(3).Replace('\','/') 
+    $dockerContext = "/mnt/$drive/$dockerContext"
```

Comment:
This could be moved to some common function:

```
function ConvertTo-DockerPath
{
    param(
        [ValidateScript({ Test-Path $_ })]
        [string]$PathToConvert
    )

    $pathInfo = Resolve-Path $PathToConvert
    $drive = $pathinfo.Drive.Name.ToLowerInvariant()
    Write-Output "/mnt/$drive/$($pathInfo.Path.Substring(3).Replace('\','/'))"  
}

$dockerDir = ConvertTo-DockerPath $dockerDir
$dockerContext = ConvertTo-DockerPath $dockerContext
```